doc/source/Overview.rst

```diff
@@ -22,9 +22,16 @@ spack-stack is maintained by:
 - Cameron Book (@ulmononian) and Mark Potts (@mark-a-potts), EPIC
 
 ===============
-Getting Started
+Getting started
 ===============
 
-Basic usage of spack-stack is described :numref:`Section %s <UsingSpackEnvironments>`. Using spack-stack requires that you have a fully configured and installed spack-stack environment. In Many of supported platforms ready-to-use spack-stack installations available locally. You can see a list of those sites and site-specific instructions and caveats in :numref:`Section %s <Preconfigured_Sites>`.
+If you wish to **access an existing spack-stack installation** on a given system, such as to load packages and compile and run a UFS or JEDI application, see the instructions in :numref:`Section %s <UsingSpackEnvironments>`. Ready-to-use spack-stack installations are available on a number of HPC and cloud platforms. A list of those platforms, along with special instructions and caveats for each, can be found in :numref:`Section %s <Preconfigured_Sites>`.
 
-If you are a developer looking to build and run software locally using spack-stack, or if you are looking to install spack-stack on a new platform you can follow the instructions in :numref:`Section %s <NewSiteConfigs>`.
+If you wish to quickly **create a new spack-stack environment (stack installation)**, either on a personal machine or on a supported platform where you are the maintainer, see :numref:`Section %s <CreatingEnvironment>`.
```

Comment:
At some point we want to refer to the "Maintainers" section that contains detailed instructions for maintainers of tier-1 platforms (or advanced users who want to replicate/update a tier-1 site config).

Reply:
Just added a reference, see what you think

Reply:
Yes, that's great, thanks very much!